### ISSUE TITLE
[Suggestion] homing like cartridges for Thaumcraft

### ISSUE THREAD

**Issue by LeoBeliik** [BluSunrize/ImmersiveEngineering#3218]
Basically add cartridges like Homing Cartridge or Crystalized Will Cartridge but for Thaumcraft and instead of terrasteel or demon will, it will be crafted with void metal :)

**Comment by EvilCooky** [comment]
Voidmetal bullets could give targets the Flux effect from Thaumcraft.
Other then that... maybe some vis crystal bullets?
But I guess that's something that would fit better into a Thaumcraft add-on Mod.

**Comment by crazytobuildmc** [comment]
I'd say:
Thaumium: slight flux, pierces enemies, ignores some armour
Voidmetal: causes large amounts of flux to the target, releases some flux into the environment, homing, sticky warp effect
Ignis: long burn
Terra: armour pierce
Aqua: causes a short suffocation debuff, dealing damage at the same rate as being trapped in a block
Aer: heavy knockback
Ordo: heals instead of doing damage, even on undead
Perditio: slight temporary warp, breaks and weathers blocks, very slight temporary warp, damages held item if it has durability

**Comment by Gammaboy45** [comment]
Voidmetal seems more fitting for, say, replenishing cartridges. The extent of research it may take for some people to reach voidmetal tier warrants a bit more than just another homing cartridge, not to mention that "homing" isn't a frequent attribute of voidmetal gear. I feel like a replenishing cartridge should have some downside, if not be more costly than an average voidmetal tool. Perhaps such crafting would be better done on Thaumcraft's end. As for flux from voidmetal, that may be a reasonable downside to discourage abundant use of the cartridge near important sites. I like the idea of primal cartridges. Perhaps the crafting could be much like tipped arrows, but using phials. Maybe shooting essentia crystals directly? As for flux damage, it'd only make sense that flux crystals then be crafted into cartridges much like primals. Thaumium is considerably stable as an alchemical product, so flux/warp from thaumium seems unfitting. Warping, however, is not an aspect; it is, though, an attribute of voidmetal. It'd make more sense that voidmetal cause warping than perditio.

So to recap my take on things:
Voidmetal cartridges: Replenishing ammunition, potential flux and/or warping effect
Primals: Perditio should not cause warping
Thaumium: Perhaps just a stronger damage cartridge, but no flux
Homing cartridges would be neat, but perhaps from another material?
